app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    data = command.split(" ")
+    if data[0] == "mv" and len(data) == 3:
```

Comment:
The condition `len(data) == 3` ensures that the command has exactly three parts. This is good for ensuring the correct format, but you might want to add a check to ensure that `data[0]` is exactly 'mv' to avoid any case sensitivity issues.